doc/source/admin/conda_faq.rst

```diff
@@ -274,7 +274,7 @@ install new tools, even if ``conda_auto_install`` is disabled.
 
 More improvements to the UI will be coming in future releases. To see if Galaxy
 has created a Trinity environment for you have a look at folder under
-``<tool_dependency_dir>/_conda/envs/``.
+``<tool_dependency_dir>/_conda/envs/``(or ``<conda_prefix>/envs`` if you bring your own Conda environment).
```

Comment:
Sure I don't mind :). As people should have read the above statements I would guess they know what it means. But the more precise the best :)